packages/gestalt/src/shared/FormLabel.js

```diff
@@ -1,19 +1,27 @@
 // @flow strict
 import { type Node as ReactNode } from 'react';
+import classnames from 'classnames';
 import styles from './FormLabel.css';
 import InternalLabel from '../Label/InternalLabel';
 import Text from '../Text';
 
+type SizeType = 'sm' | 'md' | 'lg';
+
 type Props = {
   id: string,
   label: string,
   labelDisplay?: 'visible' | 'hidden',
+  size?: SizeType,
 };
 
-export default function FormLabel({ id, label, labelDisplay }: Props): ReactNode {
+const applyDensityStyle = (size?: SizeType) => (size === 'lg' ? styles[`${size}`] : null);
```

Comment:
what is the default size?? Should we define a default one?

Reply:
Sure. See above. `md` and `sm` don't have explicit styles because they share the default class